eng/packages/http-client-csharp/generator/TestProjects/Spector/http/azure/example/basic/src/Generated/ActionRequest.Serialization.cs

```diff
@@ -0,0 +1,245 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
```

Comment:
We need to update the eng/scripts/Generate.ps1 script to remove the exclusion for azure example. We should then run this script locally and check in the code.